webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSInMemory.java

```diff
@@ -88,7 +88,13 @@ public void pull(
       Map<String, String> localeMapping,
       String skipTextUnitsWithPattern,
       String skipAssetsWithPathPattern,
-      List<String> optionList) {}
+      List<String> optionList) {
+    try {
+      Thread.sleep(60000);
+    } catch (InterruptedException e) {
+      throw new RuntimeException(e);
```

Comment:
Is this intended or is it test code that remained in the PR?

Reply:
That was test code used to delay my local third party call, will remove now